table/tables/partition.go

```diff
@@ -903,6 +903,29 @@ func (t *partitionedTable) PartitionExpr() (*PartitionExpr, error) {
 	return t.partitionExpr, nil
 }
 
+func (t *partitionedTable) GetPartitionColumnNames() []model.CIStr {
```

Comment:
Can you add a unit test for this function with various partition types?

Reply:
Done! Thank you for the suggestion.